android/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
   package="com.photoproject">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
```

Comment:
You don't need this permission for photos

Reply:
fixed